Bylaws.md

```diff
@@ -127,7 +136,9 @@ This section applies to any ballot that proposes a Final Guideline or a Final Ma
 
 ## 3.1 Associate Members
 
-The Forum may enter into associate member relationships with other organizations when the CA/Browser Forum determines that maintaining such a relationship will be of benefit to the work of the Forum. In the past, entities qualifying as Associate Members have included the ACAB’c, AICPA/CICA WebTrust Task Force, the European Telecommunications Standards Institute (ETSI), Paypal, the Internet Corporation for Assigned Names and Numbers, tScheme, the U.S. Federal PKI, and Certificate Issuers applying for membership but awaiting full qualification under Section 2.1. Participation as an Associate Member is by invitation only. In order to become an Associate Member, an organization must sign a mutual letter of intent, understanding, or other agreement and the Forum’s IPR Policy Agreement, unless this latter requirement is waived in writing by the Forum based on overriding policies of the Associate Member’s own organization IPR rules. Associate Members may attend face-to-face meetings, communicate with Voting Members on Member Mail List, and access Member Web Site. Associate Members are not entitled to vote except on special straw polls of the Forum (e.g. when selecting meeting dates, locations, etc.)
+Membership in the Forum or CWGs as an Associate Member may be by invitation when the CA/Browser Forum determines that maintaining such a relationship will be of benefit to the work of the Forum. In the past, entities qualifying as Associate Members have included the ACAB'c, AICPA/CICA WebTrust Task Force, the European Telecommunications Standards Institute (ETSI), the Internet Corporation for Assigned Names and Numbers, tScheme and the U.S. Federal PKI. In order to become an Associate Member, an organization must sign a mutual letter of intent/understanding, or other similar agreement, and the Forum's IPR Policy Agreement, unless this latter requirement is waived in writing by the Forum based on overriding policies of the Associate Member's own organization IPR rules.
```

Comment:
Given the updated, more specific category of membership represented by Associate Member in this ballot, combined with the desire expressed in recent discussions related to this ballot for Forum membership (but not Voting status) to be generally accessible, I think it would be appropriate (and a minor improvement) to remove the entire second sentence of this paragraph.

Reply:
I think this comment is based on a previous version of the ballot. The latest one includes the following language which keeps it consistent with the Probationary Member category:

```Associate Members may attend face-to-face meetings, communicate with other Members on the Member Mail List and/or Public Mail List, and access the Member Wiki Website. Associate Members may not propose or endorse ballots or take part in any form of voting except on special straw polls of the Forum or CWGs (e.g. when selecting meeting dates, locations, etc.).```

See also https://github.com/cabforum/forum/commit/207eaa4f954e9a84bea3ac61edefff622632ec67.

Does this address your concerns?

Reply:
@clintwilson, I intend to close this by EOD (Monday) assuming it addresses your concerns. If you still have concerns and feel strongly about it, please re-open.

Reply:
I'd rather not make any changes to that part of the Bylaws because it would require a larger discussion. I recall that those "mutual letter[s] of intent/understanding" have been used at least once, to establish a collaboration between CABF and ETSI. Perhaps there will be other candidate Associate Members that have bureaucracy to set up collaboration with non-incorporated entities.

Are you ok to table this for now and discuss at the next set of changes/improvements of the Bylaws? I will create a new issue so we don't forget this.

Reply:
I filed https://github.com/cabforum/forum/issues/38 so closing for now.